src/ip/QosConfig.h

```diff
@@ -61,6 +61,15 @@ namespace Ip
 namespace Qos
 {
 
+/**
+ * Specifies connection type (from/to Squid).
+ * Used for getting CONNMARK by getNfmarkFromConnection()
```

Comment:
Avoid describing how the thing is used. Focus on what it *is*. The use cases are likely to change.

For example:

```
/// possible Squid roles in connection handling
enum ConnectionDirection {
    dirAccepted, ///< accepted (from a client by Squid)
    dirOpened ///< opened (by Squid to an origin server or peer)
};
```